content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,141 @@
+# Glossary
+
+## General Blockchain Concepts
+
+### Consensus
+
+TODO: add
+
+### Blockchain
+
+TODO: add
+
+### Gossip Algorithms
+
+TODO: add
+
+### Liveness Threshold
+
+The number of malicious participants that can be tolerated before the consensus protocol stops functioning.
+
+### Nakamoto Type of Consensus
+
+TODO: introduce (for BTC)
+A consensus algorithm that provides a probabilistic safety guarantee; its decisions may revet with some probability ε.
+A permissionless protocol allowing each node to join or leave the system at any time.
+TODO: finish
+
+### Safety
+
+TODO: add
+
+## Avalanche Concepts
+
+### Ancestor Set
+
+TODO: add
+
+### Chit
+
+A transaction receives a chit (boolean value) when the majority (quorum) of queried nodes respond with an approval for that transaction.
```

Comment:
If it is a boolean value, then wherever it is used in the Avalanche specification document, we should refer to its value as true or false rather than one or zero. WDYT?

Reply:
Agree with that, its easier to understand that way. And in here you can refer as true = 1 and false = 0, WDYT?

Reply:
Yes, it makes sense, I'll change that
Thanks!

Reply:
I saw that afterwards yes, it makes sense then 👍